app/application/ports/scrapers/lms.py

```
@@ -0,0 +1,80 @@
+    @abstractmethod
+    async def scrape_notice_list(self, page_id: str) -> list[NoticeEntity]:
+        """
+        LMS에서 특정 강좌의 공지사항 목록을 스크래핑합니다.
+        """
+        ...
+
+    @abstractmethod
+    async def scrape_lecture_attendance_list(
+        self,
+        course_code: str,
+        raise_exception: bool = True,
```

Ruff (0.15.14)

[warning] Boolean-typed positional argument in function definition

(FBT001)